discord/ext/commands/core.py

```diff
@@ -1606,8 +1607,8 @@ def command(
 ) -> Callable[
     [
         Union[
-            Callable[Concatenate[CogT, ContextT, P]], Coro[T],
-            Callable[Concatenate[ContextT, P]], Coro[T],
+            Callable[[Concatenate[CogT, ContextT, P]], Coro[T]],
+            Callable[[Concatenate[ContextT, P]], Coro[T]],
```

Comment:
These brakcets are incorrect for python 3.10+, which is why I added this overload.

Reply:
Okay, if there's a purpose to it then I will definitely revert, just saw that it was raising errors in my linter.